pxtlib/docsrender.ts

```diff
@@ -249,12 +248,12 @@ namespace pxt.docs {
 
         if (currentTocEntry) {
             if (currentTocEntry.prevPath) {
-                params["prev"] = `<a href="${currentTocEntry.prevPath}" class="navigation navigation-prev " title="${'Previous page: {0}', currentTocEntry.prevName}">
+                params["prev"] = `<a href="${currentTocEntry.prevPath}" class="navigation navigation-prev " title="${currentTocEntry.prevName}">
```

Comment:
why this change?

Reply:
Look at the comma; the left side isn't doing anything so the TS compiler was throwing an error. I'm not sure what the desired behavior is, but my change shouldn't affect the current behavior.

Reply:
Oh... it is missing lf("..."). Let's keep this change. simpler.